next-level/hooks/scripts/teammate-quality-gate.sh

```
@@ -0,0 +1,42 @@
+# Teammate Quality Gate — TeammateIdle hook
+# Fires when an agent team teammate is about to go idle.
+# Enforces quality checks before allowing idle state.
+# Exit 2 = prevents idle, sends feedback to continue working.
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "$SCRIPT_DIR/utils.sh"
+
+INPUT=$(read_hook_input)
+TEAMMATE_NAME=$(json_field "$INPUT" "teammate_name")
+TEAM_NAME=$(json_field "$INPUT" "team_name")
```

Shellcheck (0.11.0)

[warning] TEAM_NAME appears unused. Verify use (or export if used externally).

(SC2034)